tw-tkms-starter/src/main/java/com/transferwise/kafka/tkms/TransactionalKafkaMessageSender.java

```diff
@@ -71,7 +71,7 @@ public void afterPropertiesSet() {
     environmentValidator.validate();
 
     for (String topic : properties.getTopics()) {
-      validateTopic(properties.getDefaultShard(), topic);
+      validateTopic(topic);
```

Comment:
I'm afraid we might have an edge case here when the topic exists, the service has DESCRIBE ACL on it, but it does not have WRITE. Topic validation will pass but produce requests will fail.

Reply:
I think we should implement topic validation via AdminClient instead.

Currently if we ask for a not existing topic, the metadata logs about it will get spammed until the producer is closed.